.pre-commit-config.yaml

```
@@ -49,8 +49,14 @@ repos:
         pass_filenames: false # Run full test suite
         stages: [pre-commit] # Only run on pre-commit
-
-
+  - repo: local
+    hooks:
+      - id: check-vscode-extension-sync
+        name: Check VSCode Extension Sync
+        entry: uv run --with-editable . scripts/check_extension_sync.py
+        language: system
+        types: [json]
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)